src/rp2_common/hardware_hazard3/include/hardware/hazard3/features.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Raspberry Pi Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef _HARDWARE_HAZARD3_FEATURES_
```

Comment:
note SDK convention is `_FOO_H` - this seems to have snuck thru in the original hazard3.h - I thought @lurch had a check for that somewhere.

Reply:
i can fix

Reply:
> I thought @lurch had a check for that somewhere.

Yeah, that's just for the board-header checking though ( [`tools/check_board_header.py`](https://github.com/raspberrypi/pico-sdk/blob/develop/tools/check_board_header.py#L326) ). I guess I could pull that out into a separate script which checks _all_ `.h` files...